spring-boot-starter-data-aerospike-example/sync/src/main/java/com/aerospike/example/sync/SyncCustomerRepository.java

```diff
@@ -1,9 +1,10 @@
 package com.aerospike.example.sync;
 
 import org.springframework.data.aerospike.repository.AerospikeRepository;
+import org.springframework.data.repository.CrudRepository;
 
 import java.util.List;
 
-public interface SyncCustomerRepository extends AerospikeRepository<Customer, String> {
+public interface SyncCustomerRepository extends AerospikeRepository<Customer, String>, CrudRepository<Customer, String> {
```

Comment:
Why extend `CrudRepository` directly?
`AerospikeRepository` extends `PagingAndSortingRepository` which extends `CrudRepository`.

Reply:
In Spring Boot 3 `PagingAndSortingRepository` extends `Repository`: https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/repository/PagingAndSortingRepository.html
Compare with older version:
https://docs.spring.io/spring-data/commons/docs/2.7.5/api/org/springframework/data/repository/PagingAndSortingRepository.html